applications/Covid Tracker/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width"> <!--to create a responsive site--> 
+        <meta name="description" content="Tracking covid cases around me"> 
+        <meta name="keywords" content="covid19 cases, tracking covid cases, covid tracker"> <!--for SEO--> 
+        <meta name="author" content="Rendell">
+        <title>Covid Tracker | Home</title>
+        <link rel="stylesheet" href="./css/style.css">
+    </head>
+    <Body>
```

Comment:
Also not a big deal, but you should use `<body>` for consistency.

Reply:
Oh yes, I get it.. will do.
Thank you